app/routes/surveys/components/surveys.css

```diff
@@ -109,6 +109,12 @@
   border: 1px solid #ccc;
 }
 
+.questionTypeSelectorBox span {
+  display: block;
```

Comment:
If you want it to be a block, it should be a `<div>`. That's the exact difference between divs and spans; spans are "inline" while divs "break" away from the content around it – implemented through `display: block`. 